assets/app/scripts/controllers/overview.js

```diff
@@ -289,26 +291,26 @@ angular.module('openshiftConsole')
         return;
       }
       // Make sure we have a build output
-      if (!build.parameters.output.to) {
+      if (!build.spec.output.to) {
         return;
       }
-      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
-        var trigger = deploymentConfig.triggers[i];
+      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
+        var trigger = deploymentConfig.spec.triggers[i];
         if (trigger.type === "ImageChange") {
           var triggerImage = trigger.imageChangeParams.from.name;
```

Comment:
what @smarterclayton said i believe, but if it's defined that way as an input, yes we'll assume latest and tack it on where necessary.  (eg if you feed in an ImageStream "foo" with no tag, we'll turn it into an ImageStreamTag of the form "foo:latest")